features/google/src/model/importers/utils/lang-constants.js

```diff
@@ -13,8 +13,19 @@ export const langConstants = {
     },
 };
 export function matchRegex(path, importer) {
+    // There can be non standard spaces in the path. Replace them with a normal space.
+    const sanitizedPath = path.replace(
+        /[\u00a0\u1680\u2000-\u200b\u202f\u205f\u3000\ufeff]/,
```

Comment:
This would only replace the first occurrence - maybe you'd want to use `/[\u00a0\u1680\u2000-\u200b\u202f\u205f\u3000\ufeff]/g` (basically adding `g` at the end for matching multiple times)? That would replace all of them in the string.

Reply:
Ah good point!